### ISSUE TITLE
History tab

### ISSUE THREAD

**Issue by philiprbrenan** [geany/geany#709]
Please consider adding a history tab which would keep a log of the edit actions performed so that one could quickly revisit areas just edited and know what changes have been made.   Thanks!

**Comment by elextr** [comment]
Agree, I have found it useful in some applications where the history is shown as part of the undo/redo actions so you know what you are about to undo/redo and maybe select to do more than one thing in the one action.
